pandas/core/arrays/arrow/extension_types.py

```diff
@@ -35,6 +35,9 @@ def __eq__(self, other):
         else:
             return NotImplemented
 
+    def __ne__(self, other) -> bool:
+        return not self.__eq__(other)
```

Comment:
```suggestion
        return not self == other
```

Based on the links I read about this, they recommended using this instead of `__eq__` (https://stackoverflow.com/questions/4352244/should-ne-be-implemented-as-the-negation-of-eq)